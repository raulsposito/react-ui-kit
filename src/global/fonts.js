import { css } from 'styled-components'

export const WEIGHT = {
  XLIGHT: 200,
  LIGHT: 300,
  REGULAR: 400,
  SEMIBOLD: 600,
  BOLD: 700,
  XBOLD: 800
}

export const SIZE = {
  XSMALL: '10px',
  SMALL: '12px',
  MEDIUM: '14px',
  LARGE: '16px',
  XLARGE: '20px',
  XXLARGE: '24px',
  H_MEDIUM: '32px',
  H_LARGE: '40px',
  H_XLARGE: '48px'
}

export const xlight = css`font-weight: ${WEIGHT.XLIGHT};`
export const light = css`font-weight: ${WEIGHT.LIGHT};`
export const regular = css`font-weight: ${WEIGHT.REGULAR};`
export const semibold = css`font-weight: ${WEIGHT.SEMIBOLD};`
export const bold = css`font-weight: ${WEIGHT.BOLD};`
export const xbold = css`font-weight: ${WEIGHT.XBOLD};`

export const fontWeight = {
  xlight,
  light,
  regular,
  semibold,
  bold,
  xbold
}

export const FONT_WEIGHT = {
  XLIGHT: xlight,
  LIGHT: light,
  REGULAR: regular,
  SEMIBOLD: semibold,
  BOLD: bold,
  XBOLD: xbold
}

export const xsmall = css`font-size: ${SIZE.XSMALL};`
export const small = css`font-size: ${SIZE.SMALL};`
export const medium = css`font-size: ${SIZE.MEDIUM};`
export const large = css`font-size: ${SIZE.LARGE};`
export const xlarge = css`font-size: ${SIZE.XLARGE};`
export const xxlarge = css`font-size: ${SIZE.XXLARGE};`

export const hmedium = css`font-size: ${SIZE.H_MEDIUM};`
export const hlarge = css`font-size: ${SIZE.H_LARGE};`
export const hxlarge = css`font-size: ${SIZE.H_XLARGE};`

export const fontSize = {
  xsmall,
  small,
  medium,
  large,
  xlarge,
  xxlarge,

  hmedium,
  hlarge,
  hxlarge
}

export const FONT_SIZE = {
  XSMALL: xsmall,
  SMALL: small,
  MEDIUM: medium,
  LARGE: large,
  XLARGE: xlarge,
  XXLARGE: xxlarge,
  HMEDIUM: hmedium,
  HLARGE: hlarge,
  HXLARGE: hxlarge
}

export const openSans = css`font-family: 'Open Sans', sans-serif;`
export const roboto = css`font-family: 'Roboto', sans-serif;`
