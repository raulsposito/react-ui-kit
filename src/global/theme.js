// https://www.color-hex.com/color/8293ee

export const colors = {
  primary: {
    regular: '#FC760D',
    dark: '#E46B0B',
    darker: '#C66F2B',
    hover: '#E17622',
    focus: '#B97641',
    selected: '#D4762C',
    disabled: '#F99546'
  },
  variant: {
    regular: '#FCB10D',
    dark: '#E29F0C',
    darker: '#C9A557',
    hover: '#E4A418',
    focus: '#BA923B',
    selected: '#D69E23',
    disabled: '#FBBF3C'
  },
  secondary: {
    regular: '#2FCFCF',
    dark: '#1F9A9A',
    darker: '#147676',
    hover: '#3DBDBD',
    focus: '#52A2A2',
    selected: '#44B4B4',
    disabled: '#5FD8D8'
  },
  success: {
    regular: '#3DB265',
    dark: '#48A668',
    darker: '#59946D',
    hover: '#48A668',
    focus: '#59946D',
    selected: '#59946D',
    disabled: '#9EBCA8'
  },
  error: {
    regular: '#FF5668',
    dark: '#E45C6B',
    darker: '#BA666F',
    hover: '#E45C6B',
    focus: '#BA666F',
    selected: '#BA666F',
    disabled: '#F8A4AC'
  },
  warning: {
    regular: '#FFAF00',
    dark: '#E5BB4D',
    darker: '#CCA644',
    hover: '#E4A418',
    focus: '#BA923B',
    selected: '#D69E23',
    disabled: '#FBBF3C'
  },
  contrast: {
    regular: '#3C3C3C',
    disabled: '#E4E4E4'
  },
  system: {
    regular: '#BAB5B4',
    dark: '#CBC6C4',
    darker: '#D7D4D1',
    hover: '#ACA8A8',
    focus: '#989595',
    selected: '#A6A2A1',
    disabled: '#C8C4C3'
  },
  background: {
    primary: '#FFFFFF',
    regular: '#FFFFFF',
    hover: '#F1F1F1',
    disabled: '#F8F8F8'
  },
  events: {
    inprogress: '#3DB265',
    next: '#5678CF',
    finished: '#BAB5B4',
    suspended: '#FFD056',
    cancelled: '#FF5668',
    white: '#FFFFFF'
  }
}

export const spacing = {
  quarter: '2px',
  half: '4px',
  threeQuarters: '6px',
  one: '8px',
  oneAndAQuarter: '10px',
  oneAndAHalf: '12px',
  two: '16px',
  twoAndAQuarter: '18px',
  twoAndAHalf: '20px',
  three: '24px',
  threeAndAHalf: '28px',
  four: '32px',
  fourAndAHalf: '36px',
  five: '40px',
  fiveAndAHalf: '44px',
  six: '48px',
  sixAndAHalf: '52px',
  seven: '56px',
  sevenAndAHalf: '60px',
  eight: '64px',
  eightAndAHalf: '68px',
  nine: '72px',
  nineAndAHalf: '76px',
  ten: '80px',
  tenAndAHalf: '84px',
  eleven: '88px',
  elevenAndAHalf: '92px',
  twelve: '96px',
  twelveAndAHalf: '100px',
  thirteen: '104px',
  fourteen: '112x',
  fifteen: '120px'
}

export const page = {
  width: '1200px'
}
