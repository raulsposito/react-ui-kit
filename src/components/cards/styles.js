import styled, { css } from 'styled-components'

import { colors, spacing } from '@global/theme'
import { column, row } from '@global/styles'

const Vertical = css`
  ${column};
  height: fit-content;
  width: 320px;
`

const Horizontal = css`
  ${row};
  width: 730px;
  height: 174px;
`

export const MyCard = styled.div`
  position: relative;
  border-radius: 5px;
  background-color: ${colors.background.regular};

  ${({ type }) => (type === 'horizontal' ? Horizontal : Vertical)};
`

export const Container = styled.div`
  ${({ type }) => (type === 'horizontal' ? row : column)};
  padding: ${({ type }) => (
    type === 'horizontal' ? `${spacing.oneAndAHalf}` : `${spacing.two} ${spacing.twoAndAHalf}`
  )};
  width: 100%;
  height: 100%;
`

export const Info = styled.div`
  ${column};
  position: relative;
  flex: 1;
  height: 100%;
`

export const MultiButtonContainer = styled.div`
  ${row};
  justify-content: flex-end;
  align-items: center;
  width: 100%;
  margin-top: auto;

  > :not(:last-child) {
    margin-right: ${spacing.three};
  }
`

export const SingleButtonContainer = styled.div`
  height: 100%;
  width: 128px;

  display: flex;
  justify-content: center;
  align-items: center;
`

const VerticalImg = css`
  width: 100%;
  height: 180px;
  border-radius: 5px 5px 0 0;
`

const HorizontalImg = css`
  width: auto;
  height: 100%;
  border-radius: 5px 0 0 5px;
`

export const Img = styled.img`
  object-fit: cover;
  ${({ type }) => (type === 'horizontal' ? HorizontalImg : VerticalImg)};
`

export const BadgeContainer = styled.div`
  position: absolute;
  top: 0;
  left: 0;

  margin-left: -${spacing.two};
  padding-top: ${spacing.twoAndAQuarter};

  > :not(:first-child) {
    margin-top: ${spacing.oneAndAQuarter};
  }
`

export const TextContainer = styled.div`
  text-overflow: ellipsis;
  overflow: hidden;

  /* stylelint-disable-next-line value-no-vendor-prefix */
  display: -webkit-box !important;
  -webkit-line-clamp: 3;
  /* stylelint-disable-next-line property-no-vendor-prefix */
  -webkit-box-orient: vertical;
  white-space: normal;
`
