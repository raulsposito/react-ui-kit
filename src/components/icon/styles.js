import styled, { css } from 'styled-components'

import { colors, spacing } from '@global/theme'

const getSize = size => {
  const s = (() => {
    switch (size) {
      case 'xsmall': return spacing.oneAndAHalf
      case 'small': return spacing.two
      case 'large': return spacing.three
      case 'xlarge': return spacing.threeAndAHalf
      case 'xxlarge': return spacing.four
      default: return spacing.twoAndAHalf
    }
  })()

  return css`
    width: ${s};
    height: ${s};
    font-size: ${s};
  `
}

const getBorderSize = size => {
  switch (size) {
    case 'xsmall': return '2px'
    case 'small': return '2px'
    case 'large': return '3px'
    case 'xlarge': return '4px'
    case 'xxlarge': return '4px'
    default: return '3px'
  }
}

const Circle = css`
  border-radius: 50%;
  box-shadow: ${({ size, color }) => {
    const borderSize = getBorderSize(size)
    const sColor = colors[color]?.regular ?? colors.contrast.regular
    return `0 0 0 ${borderSize} ${sColor}`
  }};
`

export const StyledIcon = styled.span`
  ${({ size }) => getSize(size)}
  color: ${({ disabled, color }) => {
    if (disabled) return colors[color]?.disabled ?? colors.contrast.disabled
    return colors[color]?.regular ?? colors.contrast.regular
  }};
  ${({ circle }) => circle && Circle};
`
