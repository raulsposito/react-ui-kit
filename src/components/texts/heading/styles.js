import styled, { css } from 'styled-components'

import { FONT_WEIGHT, FONT_SIZE, openSans } from '@global/fonts'
import { colors, spacing } from '@global/theme'

const StyledH = css`
  ${openSans};
  margin: 0;

  color: ${({ disabled, color }) => {
    if (disabled) return colors[color]?.disabled ?? colors.contrast.disabled
    return colors[color]?.regular ?? colors.contrast.regular
  }};

  margin-bottom: ${({ marginBottom }) => spacing[marginBottom] ?? 0};

  text-align: ${({ align }) => {
    switch (align) {
      case 'center': return 'center'
      case 'right': return 'right'
      default: return 'left'
    }
  }};

  ${({ weight }) => {
    switch (weight) {
      case 'xlight': return FONT_WEIGHT.XLIGHT
      case 'light': return FONT_WEIGHT.LIGHT
      case 'semibold': return FONT_WEIGHT.SEMIBOLD
      case 'bold': return FONT_WEIGHT.BOLD
      case 'xbold': return FONT_WEIGHT.XBOLD
      default: return FONT_WEIGHT.REGULAR
    }
  }};

  ${({ size }) => {
    switch (size) {
      case 'xsmall': return FONT_SIZE.XLARGE
      case 'small': return FONT_SIZE.XXLARGE
      case 'large': return FONT_SIZE.HLARGE
      case 'xlarge': return FONT_SIZE.HXLARGE
      default: return FONT_SIZE.HMEDIUM
    }
  }}
`

export const H1 = styled.h1`
${StyledH};
`

export const H2 = styled.h2`
${StyledH};
`

export const H3 = styled.h3`
${StyledH};
`

export const H4 = styled.h4`
${StyledH};
`

export const H5 = styled.h5`
${StyledH};
`

export const H6 = styled.h6`
${StyledH};
`
