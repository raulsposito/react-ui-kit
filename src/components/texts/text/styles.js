import styled from 'styled-components'

import { FONT_WEIGHT, FONT_SIZE, openSans } from '@global/fonts'
import { colors, spacing } from '@global/theme'

export const StyledText = styled.p`
  ${openSans};
  margin: 0;

  color: ${({ disabled, color }) => {
    if (disabled) return colors[color]?.disabled ?? colors.contrast.disabled
    return colors[color]?.regular ?? colors.contrast.regular
  }};

  margin-bottom: ${({ marginBottom }) => spacing[marginBottom] ?? 0};

  text-align: ${({ align }) => {
    switch (align) {
      case 'center': return 'center'
      case 'right': return 'right'
      default: return 'left'
    }
  }};

  ${({ weight }) => {
    switch (weight) {
      case 'xlight': return FONT_WEIGHT.XLIGHT
      case 'light': return FONT_WEIGHT.LIGHT
      case 'semibold': return FONT_WEIGHT.SEMIBOLD
      case 'bold': return FONT_WEIGHT.BOLD
      case 'xbold': return FONT_WEIGHT.XBOLD
      default: return FONT_WEIGHT.REGULAR
    }
  }};

  ${({ size }) => {
    switch (size) {
      case 'xsmall': return FONT_SIZE.XSMALL
      case 'small': return FONT_SIZE.SMALL
      case 'large': return FONT_SIZE.LARGE
      case 'xlarge': return FONT_SIZE.XLARGE
      case 'xxlarge': return FONT_SIZE.XXLARGE
      default: return FONT_SIZE.MEDIUM
    }
  }}
`
