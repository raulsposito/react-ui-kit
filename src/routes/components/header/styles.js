import styled from 'styled-components'

import { colors } from '@global/theme'

export const Head = styled.header`
  grid-area: header;
  background-color: ${colors.primary.regular};
  box-shadow: 0px 4px 4px 0px #00000040;
  z-index: 3;
`
