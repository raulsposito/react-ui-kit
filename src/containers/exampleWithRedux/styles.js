import styled from 'styled-components'

import { spacing, colors } from '@global/theme'
import { roboto, fontSize } from '@global/fonts'
import { column, row } from '@global/styles'

export const Column = styled.div`
  ${column};
  display: flex;
  flex-direction: column;

  > * {
    margin-bottom: ${spacing.two};
  }
`

export const ButtonRow = styled.div`
  ${row};
  > :first-child {
    margin-right: ${spacing.two};
  }
`

export const Input = styled.input`
  height: ${spacing.five};
  padding-left: ${spacing.one};
  width: 500px;
  ${roboto};
  ${fontSize.small};
`

export const Button = styled.button`
  background-color: ${({ color }) => (color === 'red' ? colors.error.regular : colors.primary.regular)};
  padding: ${spacing.one} ${spacing.two};
  border: none;
  outline: none;
  border-radius: 6px;
  max-width: 300px;

  cursor: pointer;

  > p {
    color: white;
  }

  :disabled {
    opacity: 0.5;
    cursor: initial;
  }
`

export const Modal = styled(Column)`
  padding: ${spacing.threeAndAHalf};
  position: fixed;
  width: 600px;
  min-height: 400px;
  background-color: white;
  align-items: center;
  justify-content: space-around;

  border-radius: 8px;

  box-shadow: 0px 10px 10px rgba(0, 0, 0, 0.2);

  > p {
    margin: auto 0;
  }

  > button {
    width: 100%;
  }
`
