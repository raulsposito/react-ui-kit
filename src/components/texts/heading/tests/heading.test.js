import React from 'react'
import { render, screen } from '@testing-library/react'

import { getComponent } from '@utils/test'

import { SIZE, WEIGHT } from '@global/fonts'
import { colors, spacing } from '@global/theme'

import Heading from '..'
import { H1, H2, H3, H4, H5, H6 } from '../styles'

describe('<Heading />', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Heading text='Goku' />)
    expect(asFragment()).toMatchSnapshot()
  })

  describe('Children & text', () => {
    it('has children', () => {
      render(<Heading>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toBeInTheDocument()
    })
    it('has prop text', () => {
      render(<Heading text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toBeInTheDocument()
    })
  })

  describe('Disabled', () => {
    it('is not disabled', () => {
      render(<Heading text='Goku' />)
      const h = screen.getByText('Goku')

      expect(h).toHaveStyle({ color: `${colors.contrast.regular}` })
    })

    it('is not disabled and have color', () => {
      render(<Heading text='Goku' color='variant' />)
      const h = screen.getByText('Goku')

      expect(h).toHaveStyle({ color: `${colors.variant.regular}` })
    })

    it('is disabled', () => {
      render(<Heading text='Goku' disabled />)
      const h = screen.getByText('Goku')

      expect(h).toHaveAttribute('disabled')
      expect(h).toHaveStyle({ color: `${colors.contrast.disabled}` })
    })

    it('is disabled and have color', () => {
      render(<Heading text='Goku' color='variant' disabled />)
      const h = screen.getByText('Goku')

      expect(h).toHaveAttribute('disabled')
      expect(h).toHaveStyle({ color: `${colors.variant.disabled}` })
    })
  })

  describe('Types', () => {
    it('should be a h1', () => {
      render(<Heading text='Goku' />)
      const h = getComponent(H1)
      expect(h).toBeInTheDocument()
    })

    it('should be a h2', () => {
      render(<Heading text='Goku' type='h2' />)
      const h = getComponent(H2)
      expect(h).toBeInTheDocument()
    })

    it('should be a h1', () => {
      render(<Heading text='Goku' type='h3' />)
      const h = getComponent(H3)
      expect(h).toBeInTheDocument()
    })

    it('should be a h1', () => {
      render(<Heading text='Goku' type='h4' />)
      const h = getComponent(H4)
      expect(h).toBeInTheDocument()
    })

    it('should be a h1', () => {
      render(<Heading text='Goku' type='h5' />)
      const h = getComponent(H5)
      expect(h).toBeInTheDocument()
    })

    it('should be a h6', () => {
      render(<Heading text='Goku' type='h6' />)
      const h = getComponent(H6)
      expect(h).toBeInTheDocument()
    })
  })

  describe('Sizes', () => {
    it('xsmall', () => {
      render(<Heading text='Goku' size='xsmall' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-size': `${SIZE.XLARGE}` })
    })

    it('small', () => {
      render(<Heading text='Goku' size='small' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-size': `${SIZE.XXLARGE}` })
    })

    it('medium', () => {
      render(<Heading text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-size': `${SIZE.H_MEDIUM}` })
    })

    it('large', () => {
      render(<Heading text='Goku' size='large' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-size': `${SIZE.H_LARGE}` })
    })

    it('xlarge', () => {
      render(<Heading text='Goku' size='xlarge' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-size': `${SIZE.H_XLARGE}` })
    })
  })

  describe('Weight', () => {
    it('xlight', () => {
      render(<Heading weight='xlight'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-weight': `${WEIGHT.XLIGHT}` })
    })

    it('light', () => {
      render(<Heading weight='light'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-weight': `${WEIGHT.LIGHT}` })
    })

    it('regular', () => {
      render(<Heading>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-weight': `${WEIGHT.REGULAR}` })
    })

    it('semibold', () => {
      render(<Heading weight='semibold'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-weight': `${WEIGHT.SEMIBOLD}` })
    })

    it('bold', () => {
      render(<Heading weight='bold'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-weight': `${WEIGHT.BOLD}` })
    })

    it('xbold', () => {
      render(<Heading weight='xbold'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'font-weight': `${WEIGHT.XBOLD}` })
    })
  })

  describe('Colors', () => {
    it('contrast', () => {
      render(<Heading text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.contrast.regular}` })
    })

    it('primary', () => {
      render(<Heading color='primary' text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.primary.regular}` })
    })

    it('secondary', () => {
      render(<Heading color='secondary' text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.secondary.regular}` })
    })

    it('variant', () => {
      render(<Heading color='variant' text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.variant.regular}` })
    })

    it('success', () => {
      render(<Heading color='success' text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.success.regular}` })
    })

    it('warning', () => {
      render(<Heading color='warning' text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.warning.regular}` })
    })

    it('error', () => {
      render(<Heading color='error' text='Goku' />)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ color: `${colors.error.regular}` })
    })
  })

  describe('Align', () => {
    it('aligns left by default', () => {
      render(<Heading>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'text-align': 'left' })
    })

    it('aligns left', () => {
      render(<Heading align='left'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'text-align': 'left' })
    })

    it('aligns right', () => {
      render(<Heading align='right'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'text-align': 'right' })
    })

    it('aligns center', () => {
      render(<Heading align='center'>Goku</Heading>)
      const h = screen.getByText('Goku')
      expect(h).toHaveStyle({ 'text-align': 'center' })
    })
  })

  describe('Margin Bottom', () => {
    it('has margin bottom 0 if no prop is passed', () => {
      render(<Heading>Spiderman</Heading>)
      const h = screen.getByText('Spiderman')
      expect(h).toHaveStyle({ 'margin-bottom': '0' })
    })

    it('has margin bottom 8px if one is passed', () => {
      render(<Heading marginBottom='one'>Spiderman</Heading>)
      const h = screen.getByText('Spiderman')
      expect(h).toHaveStyle({ 'margin-bottom': `${spacing.one}` })
    })

    it('adds a margin bottom to a value that is not valid', () => {
      render(<Heading marginBottom='fifty'>Spiderman</Heading>)
      const h = screen.getByText('Spiderman')
      expect(h).toHaveStyle({ 'margin-bottom': '0' })
    })
  })
})
