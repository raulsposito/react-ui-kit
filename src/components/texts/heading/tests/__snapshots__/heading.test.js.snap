// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should match snapshot 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdfBQB kVfTNs"
  >
    Goku
  </h1>
</DocumentFragment>
`;
